src/frequenz/client/dispatch/__main__.py

```diff
@@ -253,7 +253,9 @@ async def stream(ctx: click.Context, microgrid_id: int) -> None:
         if ctx.obj["raw"]:
             click.echo(pformat(message, compact=True))
         else:
-            print_dispatch(message)
+            msg = cast(DispatchEvent, message)
+            print(f"Dispatch {msg.event.name}")
+            print_dispatch(message.dispatch)
```

Comment:
Maybe you can use the new DispatchEvent variable `msg`, like `msg.dispatch` instead?
I also wonder if `cast` is really needed given that it seems it's already using `message` as a DispatchEvent here, or is it another type?

Reply:
It's not really needed, it was more for proper type support and editor type hint support.
I did intend to use it below though :D